src/index.html

```diff
@@ -22,6 +22,347 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        href="#"
+        class="logo"
+      >
+        <img
+          src="Images/logo.png"
+          alt="MoYo Logo"
+        />
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav_item">
+            <a
+              class="nav_link is-active"
+              href="#"
+            >
+              Apple
+            </a>
+          </li>
+
+          <li class="nav_item">
+            <a
+              class="nav_link"
```

Comment:
Class names should be written in kebab-case. The class `nav_link` should be `nav-link`.